.env.sample

```diff
@@ -0,0 +1,6 @@
+SECRET_KEY=secret
+POSTGRES_HOST=POSTGRES_HOST
+POSTGRES_PORT=POSTGRES_PORT
+POSTGRES_DB=POSTGRES_DB
```

Comment:
wait_for_db management command is not implemented

Reply:
![image](https://github.com/mate-academy/py-dockerize-cinema/assets/44442087/b797f38b-40f0-4604-94b4-56812c3ad722)